pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -496,11 +520,16 @@ const (
 	AuditPolicyMinimal     AuditPolicyPreset = "minimal"
 )
 
+// AuditLoggingSettings configures audit logging functionality.
 type AuditLoggingSettings struct {
-	Enabled      bool              `json:"enabled,omitempty"`
+	// Enabled will enable or disable audit logging.
+	Enabled bool `json:"enabled,omitempty"`
+	// Optional: PolicyPreset can be set to utilize a pre-defined set of audit policy rules.
 	PolicyPreset AuditPolicyPreset `json:"policyPreset,omitempty"`
 }
 
+// EventRateLimitConfig configures the `EventRateLimit` admission plugin.
+// Also see: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#eventratelimit
```

Comment:
Should this be "See also" instead of "Also see"?

Reply:
Yeah good point. I'll switch to "More info:", which is what the Kubernetes resources seem to use for links like that.